todo_app/app.py

```diff
@@ -1,11 +1,17 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, get_item, add_item, save_item, remove_item
```

Comment:
Very minor, but no point importing functions that you don't use
```suggestion
from todo_app.data.session_items import get_items, add_item
```

In fact, more strongly - `remove_item` doesn't exist in `session_items.py` so this actually errors and stops your app from working!